File: Answer/3512.使数组和能被-k-整除的最少操作次数.js
/*
 * @lc app=leetcode.cn id=3512 lang=javascript
 *
 * [3512] 使数组和能被 K 整除的最少操作次数
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var minOperations = function(nums, k) {
    var res = 0;
    nums.forEach(n => {
        res = (res + n) % k;
    });
    return res;
};
// @lc code=end

File: Answer/3512.使数组和能被-k-整除的最少操作次数.go
/*
 * @lc app=leetcode.cn id=3512 lang=golang
 *
 * [3512] 使数组和能被 K 整除的最少操作次数
 */

// @lc code=start
func minOperations(nums []int, k int) int {
    res := 0
	for _, n := range nums {
		res = (res + n) % k
	}
	return res
}
// @lc code=end

